src/main/java/SumOfMultipleThreeAndFiveTogether.java

```diff
@@ -0,0 +1,12 @@
+
+import java.util.stream.IntStream;
+
+public class SumOfMultipleThreeAndFiveTogether {
+
+    public static int generateSumOfMultipleThreeAndFive(int numberFromUser) {
+        return IntStream
+                .range(0,numberFromUser)
```

Comment:
To powinno być na równi z `IntStream` w linii 7 - taka konwencja że tworzenie strumienia jest zawsze w jednej linii, zaś operacje na strumieniu w nowych liniach.